lib/net.js

```diff
@@ -492,7 +492,7 @@ Object.defineProperty(Socket.prototype, 'readyState', {
 
 
 Object.defineProperty(Socket.prototype, 'bufferSize', {
-  get: function() {
+  get: function() { // eslint-disable-line getter-return
```

Comment:
FYI, `eslint-disable-line getter-return` is unnecessary if this function returns `undefined` explicitly. 

```js
  get: function() {
    if (this._handle) {
      return this[kLastWriteQueueSize] + this.writableLength;
    }
    return undefined;
  }
```

Reply:
@shisama in this case I thought it would be simplest to just add the comment. If you feel strongly about it, I can add the `return undefined;` instead.

Reply:
@cjihrig No, I am not particular about it. This is just a proposal. Thanks for your reply :)